databox.sh

```diff
@@ -69,14 +73,18 @@ case "$1" in
         export SNAPSHOT_ID=""
     fi
 
+    # If I don't specify the AMI id set it to the default databox.yml
```

Comment:
typo in the comment s\AMI id\playbook file

Reply:
thanks! Done